README.md

```diff
@@ -107,3 +107,32 @@ Meeting Agenda is maintained [here](https://github.com/redhat-developer/service-
 Please file bug reports on [Github](https://github.com/redhat-developer/service-binding-operator/issues/new). For any other questions, reach out on `service-binding-support@redhat.com`.
 
 
+## Roadmap
+
+
+**Key**
+
+⚪️ Concept validated
+
+:white_check_mark: Works, has tests and at least one documented scenario
```

Comment:
I would put the under the table.

Could you please clarify what is the difference between "concept validated" and "Works, has tests and at least one documented scenario". Should we just have a single check mark, and by positioning it in the right column we signal its maturity? Is "Beta" equal to "Technical Preview"?

Reply:
> "concept validated" 
This means that there's enough prior work done to understand that building this is possible. 


> "Works, has tests and at least one documented scenario".
Code that powers this is in master.

Reply:
> I would put the under the table.

Sure! Will do.

Reply:
Instead of introducing multiple checkmark symbols, how about that we add additional column for "conceptually validated", i.e. to name it "Planned" or "Ready for implementation"? And perhaps even link to the corresponding ehancement/github issue?

Reply:
> how about that we add additional column for "conceptually validated", i.e. to name it "Planned" or "Ready for implementation"? And perhaps even link to the corresponding enhancement/github issue?

I'm expecting that we will have two states per maturity level in many scenarios, because of newer production readiness clauses we find out. Hence, adding one new column wouldn't quite help I guess.

As an example, for "security", while an additional validating webhook is good enough in "alpha", there might be additional RBAC needs that would have to be taken are of, which will make it incorrect to be a ✅ in Beta even though it was a ✅ in Alpha :)

Reply:
> And perhaps even link to the corresponding enhancements/github issue?

( check PR description )
I was planning to have a longer doc for each feature with linked issues/PRs/proposals. What do you think?